third_party/terraform/utils/common_polling.go

```diff
@@ -0,0 +1,75 @@
+package google
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"reflect"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+type PollWaiter interface {
+	CheckResponse(map[string]interface{}, error) (wait bool, err error)
```

Comment:
It's not obvious without reading the code whether `CheckResponse` returning true means "still in progress" or "reached terminal state", could it be renamed to something that makes it super obvious which one it is?